docker-compose.elasticsearch_redis.yml

```
@@ -0,0 +1,27 @@
+    image: redis:latest
+    container_name: redis
+    ports:
+      - "6379:6379"
+    volumes:
+      - ./redis.conf:/usr/local/etc/redis/redis.conf:ro
+    command: ["redis-server", "/usr/local/etc/redis/redis.conf"]
+    restart: always
+
+
+volumes:
+  esdata01:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)